__tests__/__unit__/shared/utils.unit.test.ts

```diff
@@ -9,136 +9,127 @@
+
         return {
-            Notification: 15
+            session,
+            imperativeProfile,
+            datasetSessionNode
         };
-    });
-    beforeAll(() => {
-        Object.defineProperty(vscode.window, "showInformationMessage", {value: showInformationMessage});
-        Object.defineProperty(vscode.window, "showWarningMessage", {value: showWarningMessage});
-        Object.defineProperty(ZoweExplorerApiRegister, "getMvsApi", {value: getMvsApi});
-        Object.defineProperty(ZoweExplorerApiRegister, "getUssApi", {value: getUssApi});
-        Object.defineProperty(vscode.window, "withProgress", {value: withProgress});
-        Object.defineProperty(vscode, "ProgressLocation", {value: ProgressLocation});
+    }
 
-    });
+    afterAll(() => jest.restoreAllMocks());
 
-    afterEach(() => {
-        jest.clearAllMocks();
-    });
+    it("Testing that the add Suffix for datasets works", async () => {
+        blockMocks = createBlockMocks();
+        globals.defineGlobals("/test/path/");
 
-    it("should successfully call upload for a USS file if user clicks 'Yes'", async () => {
-        showInformationMessage.mockResolvedValueOnce("Yes");
-        withProgress.mockResolvedValueOnce(fileResponse);
-        await sharedUtils.willForceUpload(ussNode, null, null);
-        expect(withProgress).toBeCalledWith(
-            {
-                location: vscode.ProgressLocation.Notification,
-                title: "Saving file..."
-            }, expect.any(Function)
-        );
-    });
-    it("should successfully call upload for a data set if user clicks 'Yes'", async () => {
-        showInformationMessage.mockResolvedValueOnce("Yes");
-        withProgress.mockResolvedValueOnce(fileResponse);
-        await sharedUtils.willForceUpload(dsNode, null, null);
-        expect(withProgress).toBeCalledWith(
-            {
-                location: vscode.ProgressLocation.Notification,
-                title: "Saving data set..."
-            }, expect.any(Function)
-        );
-    });
-    it("should cancel upload if user clicks 'No'", async () => {
-        showInformationMessage.mockResolvedValueOnce("No");
-        await sharedUtils.willForceUpload(dsNode, null, null);
-        expect(showInformationMessage.mock.calls[1][0]).toBe("Upload cancelled.");
-    });
-    it("should display specific message if Theia is detected", async () => {
-        Object.defineProperty(globals, "ISTHEIA", {value : true});
-        showInformationMessage.mockResolvedValueOnce("No");
-        await sharedUtils.willForceUpload(dsNode, null, null);
-        expect(showWarningMessage.mock.calls[0][0]).toBe("A merge conflict has been detected. Since you are running inside Theia editor, a merge conflict resolution is not available yet.");
+        let node = new ZoweDatasetNode("AUSER.TEST.JCL(member)", vscode.TreeItemCollapsibleState.None, blockMocks.datasetSessionNode, null);
```

Comment:
I feel like these should really be separate tests, but lets leave it as it is for now so that we can move on.